src/uikit.cs

```diff
@@ -18276,4 +18420,16 @@ interface UIFontMetrics {
 		nfloat GetScaledValue (nfloat value, [NullAllowed] UITraitCollection traitCollection);
 #endif // !WATCH
 	}
+
+	[Static]
+	[iOS (12,0), TV (12,0), Watch (5,0)]
+	interface UIKitVersion {
+
+		[Field ("UIKitVersionNumber")]
+		double UIKitVersionNumber { get; }
+
+		[Field ("UIKitVersionString")]
+		NSString UIKitVersionString { get; }
+
+	}
```

Comment:
remove those and put them in `UIKit.ignore`
we're doing the same in other semi-versioned frameworks